### PR TITLE
python3-jeepney: update to 0.6.0

### PR DESCRIPTION
**Pull request by ndowens**
Add tests

### CODE REVIEW

File: srcpkgs/python3-jeepney/template [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
 wrksrc="jeepney-${version}"
 build_style=python3-module
 hostmakedepends="python3 python3-setuptools"
 depends="python3"
+checkdepends="python3-pytest python3-testpath"
 short_desc="Low-level, pure Python DBus protocol wrapper"
 maintainer="Orhpaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="https://gitlab.com/takluyver/jeepney"
+changelog="https://gitlab.com/takluyver/jeepney/-/raw/${version%.*}/docs/release-notes.rst"
```
Comment (ericonr) [inline srcpkgs/python3-jeepney/template:14]: Please use version from master.

Comment (ndowens) [reply srcpkgs/python3-jeepney/template:14]: :) Just say master if you find more, I forgot which all I changed them on. Save you some typing :D

**Comment by ericonr** [comment]
Can you make sure `python3-SecretStorage` works fine with this new version?

**Comment by ndowens** [comment]
> Can you make sure `python3-SecretStorage` works fine with this new version?

Following an example on how to use SecretStorage, https://secretstorage.readthedocs.io/en/latest/, works fine

**Comment by ericonr** [comment]
Thanks!

**Pull request merged**